### ISSUE TITLE
Giant Mass Fab Rework 

### ISSUE THREAD

**Issue by ScriptedPiky** [alkcorp/GTplusplus#683]
As posted from the GT++ Discord beta channel:

Giant mass fab should be rewritten from the ground up if possible, or majorly changed. Since whatever the current code or recipe maps are, it is wacky and or buggy as fuck. My guesses are that if the recipes are created as new ones and simplified that even my dumb dumb brain can process:
Parallel processing of 2 or 3 per level. 5% EU discount (max amount of EU of one amp -5%). Outputs pollution at 500 every second. The tier that it should be obtainable is either LuV or ZPM or UV. 

**UUM mode 1#**: 140 second base UUM recipe at ULV, 35 seconds with UUA inserted. Circuit 20 is required to run regular UUM. Circuit 21 is required to run WITH UUA. Circuits are inserted into the multiblock GUI (if possible) or just inserted into an input bus
**UUA mode 2#**: ?? second base UUA recipe at ULV (I forgot the actual UUA recipe timing). Doesn't need a circuit, only needs scrap or scrap boxes
**Recycler mode 3#**: Based on recycler recipe map. Literally eats anything inside the input bus as if it were a recycler. Recycler mode also gains an additional 2x the parallel bonus

**Comment by GTNH-Colen** [comment]
Agreed that this multi needs a complete overhaul, been playing with it myself in a testing world and it seems totally out of whack.

**Comment by chochem** [comment]
To circumvent the harsh efficiency loss from overclocking, I have an alternative proposal:

The main thing about uum production in gt5u is always that the EU/uum ratio is fixed for all (single block) machines. The main problem of the single block mass fabs is that the 2x/2x (hardcoded) overclocking leads to high tier machines only taking 1/8 Amp (in the case of ZPM) of their tier etc.

So a good fitting multi could be one that keeps the EU/uum ratio but allows for higher Amp usage through parallelization.

Detailed proposal:  min tier 4 (EV), upto 2^(tier -4) parallel processes, uses the single block recipe of its tier as base recipe, no EU discount or speed bonus.

Result: Those numbers would mean that a multi of any tier EV or higher can use upto a full Amp of its tier and converts that power into uum at the fixed gt5u ratio of 823,040 EU/L or 205,760 EU/L with UUA. The parallelization is needed to not hit the 1 tick limit too quickly.

Example: ZPM, 1A provided, UUM mode. single block recipe takes 2.5s/L and 16384EU/t. tier=7, so 2^(7-4)=8 processes in parallel. So overall 2.5s for 8L at 131072EU/t preserving the 823,040 EU/L.

feel free to add pollution/require a coolant or stuff like that.
